### ISSUE TITLE
letsencrypt-auto doesn’t work when symlinked

### ISSUE THREAD

**Issue by dboulet** [certbot/certbot#1698]
I tried creating a symbolic link to letsencrypt-auto in `~/.local/bin/`, but get an error message when I run it:

> Could not open requirements file: [Errno 2] No such file or directory: '/home/danboulet/.local/bin/py26reqs.txt'

Looks like the script doesn’t follow symlinks when searching for other files within its own directory.

**Comment by rask** [comment]
Same problem here.

```
$ ln -s /home/user/letsencrypt/letsencrypt-auto /home/user/bin/letsencrypt
$ which letsencrypt 
/home/user/bin/letsencrypt
$ letsencrypt --help
Cannot find the letsencrypt bootstrap scripts in /home/user/bin/bootstrap
```

Running on an up-to-date Ubuntu 14.04. The `letsencrypt` installation was cloned from this repo 5 minutes ago.

**Comment by pde** [comment]
Looks like the value of LEA_PATH in the script doesn't follow symlinks.  Probably we can do test -L on that, and if it's a symlink, apply a POSIX recipe [like this](https://groups.google.com/forum/#!topic/comp.unix.shell/3s8cuwcVnTk).

A fix should be drafted on top of #1665, I think.

**Comment by schoen** [comment]
It seems to me that this exact problem no longer exists, yet there is an equally serious problem when trying to do this nowadays which I've reported at #6394.
